### ISSUE TITLE
About the test setting of the trained model

### ISSUE THREAD

**Issue by DXQer** [ando-khachatryan/HiDDeN#22]
Thanks for your contribution of this repository. 
I have a doubt. In the original paper, the authors mentioned that the experimental results given are all tested under real attacks, such as JPEG compression with different quality factors, but in this repository, it seems that the trained models are verified and tested on the basis of simulated noise attacks?

**Comment by ando-khachatryan** [comment]
Yes, you are right, this repo only reproduces the end-to-end training and validation. Real jpeg, Digimarc, and other experiments are outside the scope of this repo.

**Comment by DXQer** [comment]
Thanks for your answers, I want to know whether you will provide the corresponding code files for testing under the real JPEG compression attack in the future.
